File: Java/Program/Tabel.java
public class Tabel {
    private int baris;
    private int kolom;

    Tabel() {

    }

    //Constructor 
    Tabel(int baris, int kolom){
        this.baris = baris;
        this.kolom = kolom;
    }

    //Getter dan setter untuk baris
    public int getBaris(){
        return baris;
    }

    public void setBaris(int baris){
        this.baris = baris;
    }

    //Getter dan setter untuk kolom
    public int getKolom(){
        return kolom;
    }

    public void setKolom(int kolom){
        this.kolom = kolom;
    }

    //Method untuk menampilkan list dalam bentuk tabel
    public void buatBaris(int baris, int kolom, String[][] data){
        int i = 0, j = 0;
        int max[] = new int[kolom];
        for (i = 0; i < kolom; i++) {
            max[i] = 0;
            for (j = 0; j < baris; j++) {
                if (data[j][i].length() > max[i]) {
                    max[i] = data[j][i].length();
                }
            }
        }

        for (i = 0; i < baris; i++) {
            for (j = 0; j < kolom; j++) {
                for (int k = 0; k < max[j]; k++) {
                    System.out.print("-");
                }
                System.out.print("--");
            }
            System.out.println("---------");
            for (j = 0; j < kolom; j++) {
                System.out.print("|" + data[i][j]);
                for (int k = 0; k < max[j] - data[i][j].length(); k++) {
                    System.out.print(" ");
                }
                System.out.print("  ");
            }
            System.out.println("|");
        }
        for (j = 0; j < kolom; j++) {
            for (int k = 0; k < max[j]; k++) {
                System.out.print("-");
            }
            System.out.print("--");
        }
        System.out.println("---------");
    }
}

File: Pyhton/Program/Tabel.py
class Tabel:
    def __init__(self):
        self.baris = 0
        self.kolom = 0

    def __init__(self, baris, kolom):
        self.baris = baris
        self.kolom = kolom

    # Getter dan setter untuk baris
    def getBaris(self):
        return self.baris

    def setBaris(self, baris):
        self.baris = baris

    # Getter dan setter untuk kolom
    def getKolom(self):
        return self.kolom

    def setKolom(self, kolom):
        self.kolom = kolom

    # Method untuk menampilkan list dalam bentuk tabel
    def buatTabel(self, data):
        max_len = [0] * self.kolom
        for i in range(self.kolom):
            for j in range(self.baris):
                max_len[i] = max(max_len[i], len(data[j][i]))

        for i in range(self.baris):
            for j in range(self.kolom):
                for k in range(max_len[j]):
                    print("-", end="")
                print("--", end="")
            print("---------")
            for j in range(self.kolom):
                print("|" + data[i][j], end="")
                print(" " * (max_len[j] - len(data[i][j])) + "  ", end="")
            print("|")
        for j in range(self.kolom):
            for k in range(max_len[j]):
                print("-", end="")
            print("--", end="")
        print("---------")
